docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrate Kyma with AWS CloudWatch
+
+## Overview
+
+The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. The following example outlines how to integrate with [AWS CloudWatch](https://aws.amazon.com/cloudwatch) as a backend. Because CloudWatch does not support OTLP ingestion natively, you must deploy the [AWS Distro for OpenTelemetry](https://aws-otel.github.io) additionally.
+
+![overview](../assets/cloudwatch.drawio.svg)
```

Comment:
I would rename the labels for the exporters / output from "AWS Logs / Metrics / Traces" to "CloudWatch Logs / Metrics" and "X-Ray Traces"